plugins/out_pgsql/pgsql.c

```diff
@@ -168,6 +168,15 @@ static int cb_pgsql_init(struct flb_output_instance *ins,
         ctx->cockroachdb = FLB_FALSE;
     }
 
+    /* Schema Support */
+    tmp = flb_output_get_property("schema", ins);
+    if (tmp) {
+        ctx->schema = flb_sds_create(tmp);
+    }
+    else {
+        ctx->schema = FLB_PGSQL_SCHEMA
```

Comment:
Since schema is a pointer, set it to null should be enough, no need to use space in a real char here